cmd/lakectl/cmd/doctor.go

```diff
@@ -20,6 +22,12 @@ type CredentialsError struct {
 	Details string
 }
 
+var accessKeyRegexp = regexp.MustCompile(`^AKIA[I|J][A-Z0-9]{14}Q$`)
+
+func IsValidAccessKeyID(accessKeyID string) bool {
```

Comment:
Can we put the IsValid functions together?
Also do the need to be public?

Reply:
I missed this one - at first I moved it to the only place it was used.
But I've checked the implementation and the lakeFS's is using pkg/auth/service.go version of this function which just verify min/max length.
So I'm not sure we need to continue to use this one.